test/hello.test.ts

```diff
@@ -0,0 +1,10 @@
+import { describe, it } from 'mocha'
+import expect from 'expect'
+
+describe('this test', () => {
+    it('says hello', () => {
+        const text = 'hello world'
+
+        expect(text).toEqual('hello world')
+    })
+})
```

Comment:
Is this test needed? It looks like its a canary test to ensure that tests are running?

Reply:
Yeah, I remove an unneeded test, and put back in this default test as a stub. I thought for a few hours about how we might test this code, but in the end, I just left this stub as a reminder that we definitely need tests or some sort.